src/main/java/com/amazonaws/kinesisvideo/utils/AwsV4Signer.java

```diff
@@ -33,116 +27,280 @@
 @SuppressWarnings({"SpellCheckingInspection", "SameParameterValue"})
 public class AwsV4Signer {
 
-    private static final Logger logger = LoggerFactory.getLogger(AwsV4Signer.class);
-
-    private static final String ALGORITHM_AWS4_HMAC_SHA_256 = "AWS4-HMAC-SHA256";
-    private static final String AWS4_REQUEST_TYPE = "aws4_request";
-    private static final String SERVICE = "kinesisvideo";
-    private static final String X_AMZ_ALGORITHM = "X-Amz-Algorithm";
-    private static final String X_AMZ_CREDENTIAL = "X-Amz-Credential";
-    private static final String X_AMZ_DATE = "X-Amz-Date";
-    private static final String X_AMZ_EXPIRES = "X-Amz-Expires";
-    private static final String X_AMZ_SECURITY_TOKEN = "X-Amz-Security-Token";
-    private static final String X_AMZ_SIGNATURE = "X-Amz-Signature";
-    private static final String X_AMZ_SIGNED_HEADERS = "X-Amz-SignedHeaders";
-    private static final String NEW_LINE_DELIMITER = "\n";
-    private static final String DATE_PATTERN = "yyyyMMdd";
-    private static final String TIME_PATTERN = "yyyyMMdd'T'HHmmss'Z'";
-
-    private static final String METHOD = "GET";
-    private static final String SIGNED_HEADERS = "host";
+    static final String ALGORITHM_AWS4_HMAC_SHA_256 = "AWS4-HMAC-SHA256";
```

Comment:
is this for visibility for unit tests?

Reply:
Yes

Reply:
We can move these to a separate constants class.

Reply:
Done in [e2765e7](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-android/pull/92/commits/e2765e7226053c611138fb9fcb1f1e4c7c42b072)